src/services/shopify-client.test.ts

```diff
@@ -0,0 +1,162 @@
+import { AxiosStatic } from 'axios';
+import { HttpShopifyClient, SubscriberListState } from './shopify-client';
+import { RefObject } from 'react';
+import { AppSession } from './app-session';
+import { DopplerLegacyUserData } from './doppler-legacy-client';
+
+function createHttpShopifyClient(axios: any) {
+  const axiosStatic = {
+    create: () => axios,
+  } as AxiosStatic;
+  const connectionDataRef = {
+    current: {
+      status: 'authenticated',
+      jwtToken: 'jwtToken',
+      userData: {} as DopplerLegacyUserData,
+    },
+  } as RefObject<AppSession>;
+  const baseUrl = 'http://shopify.test';
+  const shopifyClient = new HttpShopifyClient({ axiosStatic, baseUrl, connectionDataRef });
+  return shopifyClient;
+}
+
+const connectedWithListReadyResponse = {
+  data: [
+    {
+      connectedOn: '2019-08-09T15:13:52.262Z',
+      dopplerListId: '27311899',
+      dopplerListName: 'Shopify Contacto',
+      importedCustomersCount: '6',
+      shopName: 'dopplerdevshop.myshopify.com',
+      shopifyAccessToken: '2741255a41f37341d0fa6d64d58e5c86',
+      syncProcessDopplerImportSubscribersTaskId: 'import-100544891',
+      syncProcessInProgress: 'false',
```

Comment:
Nice! 

Could you also add a test with a boolean false value?

It would be to prevent bad behaviors when the response is fixed.

Reply:
Sure.

Reply:
@andresmoschini Done!